woocommerce/woocommerce
# Changes to IAM footer

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:
Improve footer design and content as mentioned in [issue](https://github.com/Automattic/woocommerce.com/issues/21574)





Closes https://github.com/Automattic/woocommerce.com/issues/21574 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out this branch
2. Go to /plugins/woocommerce and run pnpm --filter='@woocommerce/plugin-woocommerce' build
3. Visit the [in-app marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions).
4. Now view the screen with less products one could be business services or search for random text like `sadasd` on search
<img width="1909" alt="Screenshot 2024-09-12 at 3 56 03 PM" src="https://github.com/user-attachments/assets/46c2d466-39f4-4263-8dbd-e3cb34873ea7">

5. Confirm footer now contained in a box similar to [design your own](https://href.li/?https://prnt.sc/6idFn5AuP3Ov) banner in Themes
6. Confirm the `woo marketplace` content on footer is now removed
7.  Also confirm `Support the ecosystem` content is added to the footer as the 4th element.
8. Test on different screen sizes and confirm nothing is broken on footer.



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Update footer design, add one more element to footer content and remove `woo-marketplace` copy at footer bottom.
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
